.pre-commit-config.yaml

```diff
@@ -20,7 +20,7 @@ repos:
     hooks:
       - id: black
   - repo: https://gitlab.com/pycqa/flake8/
-    rev: 3.9.0
+    rev: 3.9.1
```

Comment:
It is not part of homework.  You don't need to show these changes to me, please remove them from PR.  

Technical things could be merged without involving me.